understanding/20/reading-level.html

```diff
@@ -169,78 +160,43 @@ <h2>Benefits of Reading Level</h2>
    <section id="examples">
       <h2>Examples of Reading Level</h2>
       
-      
-      <ol>
+      <ul>
```

Comment:
this was changed to a `ul`, but the items within are `dt` and `dd`s.

Reply:
oops, thanks for spotting this (and the other one). clearly a find/replace across all files that went wrong (caused by the infurtiatingly inconsistent structure of the original files). if you do spot any more, shout, and I'll do a big cleanup commit on this PR